Sources/Controllers/2.0/OAFavoriteListViewController.mm

```diff
@@ -600,13 +603,14 @@ - (void)setupColor
         [app saveFavoritesToPermamentStorage];
     }
     
+    [_selectedUnsortedItems removeAllObjects];
     [self editButtonClicked:nil];
```

Comment:
Needs refactoring. We need two new methods - startEditing / finishEditing. These methods should be called from editButtonClicked. [_selectedUnsortedItems removeAllObjects] should be in finishEditing. Call finishEditing instead of [self editButtonClicked:nil].